lib/net/ftp.rb

```diff
@@ -563,6 +563,7 @@ def makepasv # :nodoc:
     def transfercmd(cmd, rest_offset = nil) # :nodoc:
       if @passive
         host, port = makepasv
+        succeeded = false
```

Comment:
This line is not actually needed.  However, I have no problem with it if you using it to be explicit.

Reply:
Thanks for your comment.

I'd like to initialize `succeeded` to clarify the intention.